### ISSUE TITLE
CSI nodes missing in ZeroW and 3B+ DeviceTrees

### ISSUE THREAD

**Issue by lurch** [raspberrypi/linux#2998]
It looks like this change https://github.com/raspberrypi/linux/commit/f288b3a9788e7d3f182af279efda84855ead747f accidentally missed out updating `bcm2835-rpi-zero-w.dts` and `bcm2837-rpi-3-b-plus.dts`

And just out of curiosity, how does this work for the original variant of PiZero which didn't have a CSI connector?

**Comment by 6by9** [comment]
Yes, they both need to include the bcm283x-rpi-csi1-2lane.dtsi file.

This is only relevant if trying to connect a sensor or other CSI2 device to be controlled by the ARM, and there aren't many that we support.
Yes, in theory you can configure the early Pi0 revision which had no camera connector on to try talking to a sensor. It'll fail to probe. Checking the firmware, there is no specific configuration there for the Pi0 1.2, therefore all Pi0's will think they potentially can connect a camera.
